components/builder-web/app/shared/github-repo-picker/GitHubRepoPickerComponent.ts

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) 2016-2017 Chef Software Inc. and/or applicable contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Component, Input, OnInit, Output, EventEmitter } from "@angular/core";
+import { List, Map, OrderedSet } from "immutable";
+import { AppStore } from "../../AppStore";
+import { GitHubRepo } from "../../github/repo/shared/github-repo.model";
+import {fetchGitHubOrgs, fetchGitHubRepos,
+  onGitHubOrgSelect, setSelectedGitHubOrg, resetRedirectRoute} from "../../actions/index";
+
+@Component({
+  selector: "hab-github-repo-picker",
+  template: `
+  <div class="hab-github-repo-picker">
```

Comment:
Could you move this file into it's own html file? https://angular.io/guide/styleguide#extract-templates-and-styles-to-their-own-files